charts/oidc-proxy/values.yaml

```diff
@@ -52,6 +52,7 @@ service:
 ingress:
   enabled: true
   annotations: {}
+  domainName: ""
```

Comment:
Maybe comment what that is needed and empty?

Reply:
I had the same thought!

Reply:
Without having to goto the readme you can, as far as why "" it just seems to be the communal norm.

```helm show values datarepo-helm/datarepo-api```